OSDOCS-8487 update installer RN tech preview table (#71794)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

|====
|Feature |4.13 |4.14 |4.15

> |Installing {product-title} on Oracle Cloud Infrastructure with VMs

Reviewer: ```suggestion
|Installing {product-title} on {oci-first} with VMs
```

|N/A
|Developer Preview
|Technology Preview

> |Installing {product-title} on Oracle Cloud Infrastructure on baremetal

Reviewer: ```suggestion
|Installing {product-title} on {oci} on baremetal
```
Or if you'd prefer to have the Oracle name fully written out in this case for better clarity, you could probably use `{oci-first}` again.

> |Installing {product-title} on Oracle Cloud Infrastructure on baremetal

Reviewer: ```suggestion
|Installing {product-title} on Oracle Cloud Infrastructure on bare metal
```
